SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -64,6 +64,11 @@ To run a cloud deployment, you need to prepare your server-worker and client-wor
 
 ### Congratulations!
 
-You've successfully set up and launched the Starter Project and the GDK! To start developing, we recommend following our Multiserver Shooter tutorial where you can try out the GDK’s development experience by adding a new feature to the Starter Project and test it across two servers.
+You've successfully set up and launched the Starter Project and the GDK! If you haven't already, we recommend following our Multiserver Shooter tutorial where you can try out the GDK’s development experience by adding a new feature to a project and test it across two servers.
+
+If you have an existing Unreal multiplayer project, you can follow our detailed [porting guide]({{urlRoot}}/content/get-started/porting-unreal-project-to-gdk.md) to get it onto the GDK!
+
+You are now ready to add multiserver functionality on top of the Starter Project. Check out the documentation on [cross-server RPCs]({{urlRoot}}/content/cross-server-rpcs), [handover]({{urlRoot}}/content/handover-between-server-workers) and [singletons]({{urlRoot}}/content/singleton-actors).
+
+We will be releasing more tutorials and examples as the GDK matures. Stay tuned!
```

Comment:
Same on the "!". Docs don't get excited. We are reliable and sensible.

Reply:
#personalitygoals

Reply:
Although there is already some excitement in this doc. See "Next Steps!". I assume you'd want them removed too.

Reply:
DId *I* write "Next Steps!"?? I suspect that was a bit of Ernie text which slipped in. :-)

Docs !<3 exclamation marks.